tests/user.test.js

```diff
@@ -1,7 +1,5 @@
-'use strict';
-
-const supertest = require('supertest');
-const { createServer } = require('../src/createServer');
+import supertest from 'supertest';
+import { createServer } from '../src/createServer';
```

Comment:
I don't understand why you need it if you use CommonJS

Reply:
You're right, that's just a leftover from previous commit where I tried to use imports.